pyrolite/util/classification.py

```diff
@@ -455,10 +455,11 @@ def add_to_axes(
                         xy=(x, y),
                         ha="center",
                         va="center",
+                        fontsize=kwargs.get("fontsize", 8),
```

Comment:
Probably also a way to pass this via `subkwargs` below, but this works and is transparent!